src/acl/AtStepData.cc

```diff
@@ -30,9 +37,9 @@ ACLAtStepData::~ACLAtStepData()
 }
 
 bool
-ACLAtStepData::match(Ssl::BumpStep  toFind)
+ACLAtStepData::match(XactionSteps toFind)
 {
-    for (std::list<Ssl::BumpStep>::const_iterator it = values.begin(); it != values.end(); ++it) {
+    for (auto it = values.cbegin(); it != values.cend(); ++it) {
```

Comment:
Consider using std::find() or, if we cannot use that existing/standard function, a simpler for-range loop:

```suggestion
    for (const auto value: values) {
```

Reply:
commit 4257238db7